execution-engine/contracts/examples/simple-token/src/lib.rs

```diff
@@ -0,0 +1,117 @@
+#![no_std]
+
+extern crate alloc;
+
+use alloc::{collections::BTreeMap, string::String};
+
+use contract::{
+    contract_api::{runtime, storage, TURef},
+    unwrap_or_revert::UnwrapOrRevert,
+};
+
+use types::{ContractRef, Key, CLValue, U512, ApiError};
+use core::convert::TryInto;
+
+const CONTRACT_NAME: &str = "prac_token";
+const CONTRACT_EXT_NAME: &str = "prac_token_ext";
+const CONTRACT_PROXY_NAME: &str = "prac_token_proxy";
+const TOKEN_MAP_NAME: &str = "token_map";
+const TOTAL_SUPPLY_NAME: &str = "total_supply";
+
+fn init_token_db(){
+    // Token value storage
+    let init_token_map: BTreeMap<Key, U512> = BTreeMap::new();
+    let token_map_turef: TURef<BTreeMap<Key, U512>> = storage::new_turef(init_token_map);
```

Comment:
I think you need to store the map through `store_function` or `store_function_at_hash`.